common/src/main/java/com/wynntils/models/players/hades/HadesClientHandler.java

```diff
@@ -30,6 +30,8 @@ public class HadesClientHandler implements IHadesClientAdapter {
     public HadesClientHandler(HadesConnection hadesConnection, HadesUserRegistry userRegistry) {
         this.hadesConnection = hadesConnection;
         this.userRegistry = userRegistry;
+
+        Thread.currentThread().setContextClassLoader(WynntilsMod.class.getClassLoader());
```

Comment:
Please state that this is a workaround due to a bug in the forge event bus, and link to `https://github.com/MinecraftForge/EventBus/issues/44`.

(The github gui allows me to make suggestions to lines, but not insert new lines, otherwise I'd fixed it myself)